apps/site/components/Common/Button/index.tsx

```diff
@@ -24,6 +24,7 @@ const Button: FC<ButtonProps> = ({
     href={disabled ? undefined : href}
     aria-disabled={disabled}
     className={classNames(styles.button, styles[kind], className)}
+    tabIndex={!href ? 0 : undefined}
```

Comment:
Why is this href dependent

Reply:
Because if there is an href, this property is not necessary, as the href already makes it accessible via the keyboard

ref: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex